dsp-role/repo/src/main/scala/dsp/role/repo/impl/RoleRepoLive.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.role.repo.impl
+
+import dsp.role.api.RoleRepo
+import dsp.role.domain.Role
+import dsp.valueobjects.Id.RoleId
+import zio._
+import zio.stm.TMap
+
+import java.util.UUID
+
+/**
+ * Role repo live implementation.
+ *
+ * @param roles a map of roles
+ */
+final case class RoleRepoLive(
+  roles: TMap[UUID, Role]
+) extends RoleRepo {
+
+  /**
+   * @inheritDoc
+   */
+  override def storeRole(r: Role): UIO[RoleId] =
+    (for {
+      _ <- roles.put(r.id.uuid, r)
+    } yield r.id).commit.tap(_ => ZIO.logInfo(s"Stored role with ID: ${r.id.uuid}"))
+
+  /**
+   * @inheritDoc
+   */
+  override def getRoles(): UIO[List[Role]] =
+    roles.values.commit.tap(rolesList => ZIO.logInfo(s"Found roles number: ${rolesList.size}"))
+
+  /**
+   * @inheritDoc
+   */
+  override def getRoleById(id: RoleId): IO[Option[Nothing], Role] =
+    roles
+      .get(id.uuid)
+      .commit
+      .some
+      .tapBoth(
+        _ => ZIO.logInfo(s"Role with ID: ${id.uuid} not found"),
```

Comment:
```suggestion
        _ => ZIO.logInfo(s"Role with ID '${id.uuid}' not found"),
```
I don't think the colon in the middle of the word works nicely... maybe use single quotes instead?